### ISSUE TITLE
Link actions together

### ISSUE THREAD

**Issue by ufx** [ufx/GarlandTools#9]
Actions often contain references to other actions.  Do whole word detection and link them.

**Comment by nakomaru** [comment]
Consider doing the same for achievements, such as "A Life of Adventure II".

http://www.garlandtools.org/db/#achievement/1190

**Comment by ufx** [comment]
Definitely.  Achievements should be easy as the tables already link between each other.
